python/server/PythonServerWrapper.cpp

```diff
@@ -875,6 +875,19 @@ namespace {
         fleet->SetAggressive(fleet->HasArmedShips());
         ship->SetFleetID(fleet->ID());
 
+        // set the meters of the ship to max values
+        ship->ResetTargetMaxUnpairedMeters();
+        ship->ResetPairedActiveMeters();
+
+        ship->GetMeter(METER_MAX_FUEL)->SetCurrent(Meter::LARGE_VALUE);
+        ship->GetMeter(METER_MAX_SHIELD)->SetCurrent(Meter::LARGE_VALUE);
+        ship->GetMeter(METER_MAX_STRUCTURE)->SetCurrent(Meter::LARGE_VALUE);
+        ship->GetMeter(METER_FUEL)->SetCurrent(Meter::LARGE_VALUE);
+        ship->GetMeter(METER_SHIELD)->SetCurrent(Meter::LARGE_VALUE);
+        ship->GetMeter(METER_STRUCTURE)->SetCurrent(Meter::LARGE_VALUE);
+
+        ship->BackPropegateMeters();
+
```

Comment:
I wonder if that block shouldn't be moved into a new member function for the ship class? `SetShipMetersToMax` or something like that. I guess very similar pieces of code are already present in other code sections which create ships, and could probably also set to use that new member function instead of duplicating these lines at several places in the codebase.